samples/apps/copilot-chat-app/webapi/Hubs/ChatHub.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.AspNetCore.SignalR;
+
+namespace SemanticKernel.Service.SignalR.Hubs;
+
+public class ChatHub : Hub
+{
+    public async Task SendMessageAsync(string user, string message)
+    {
+        await this.Clients.All.SendAsync("SendMessage", user, message);
+        Console.WriteLine("Log SendMessage: {0}", message);
```

Comment:
You probably never want to use Console.WriteLine() in a server / service.  Use ILogger<ChatHub> instead - It can be provided by dependency injection through a constructor to ChatHub.